src/clients/node/scripts/build.sh

```diff
@@ -2,7 +2,7 @@
 
  set -e
 
-docker run -v "$(pwd)/../../..":/wrk -w /wrk/src/clients/node --entrypoint bash node:18 -c "
```

Comment:
I wonder what we should pick here?

- on the one hand, we are shipping a library, not an app, so it's advantegous to make this a rolling versions to catch cases where npm regresses us
- on the other hand, if we are doign that, we probably want latest node, rather than just lts
- on the third hand, this stuff is for _building_ the package for publishing, so arguably only we should worry about this, and not our users, and it might be better to pin?

Reply:
Eventually we probably need a matrix of versions. So I don't think it really matters what we pick at the moment.